### PR TITLE
Requested Test Added

### PR DESCRIPTION
**Pull request by AliRKat**
Session metric override test added.

Changelog entry for metric override added.

overridenMetrics field in MetricHelper changed into internal, which was causing a compiler error while it was private.

### CODE REVIEW

**Comment by AliRKat** [comment]
[[Unity] Add overriden metric test for the session request.](https://trello.com/c/63SJn0wC/1089-unity-add-overriden-metric-test-for-the-session-request)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## XX.X.X
+* Added MetricHelper class and "overriddenMetrics" field in the configuration object, allowing to pass custom or overridden metrics.
```
Comment (ArtursKadikis) [inline CHANGELOG.md:2]: reword this and don't mention metric helper

File: Assets/Plugins/CountlySDK/Models/CountlyConfiguration.cs [hunk 2]
```diff
@@ -237,10 +236,5 @@ public void AddNotificationListener(INotificationListener listener)
         {
             NotificationEventListeners.Add(listener);
         }
-        
-        public void SetMetricOverride(Dictionary<string, string> overridenMetrics)
```
Comment (ArtursKadikis) [inline Assets/Plugins/CountlySDK/Models/CountlyConfiguration.cs:241]: add this back in

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
use the setter for the tests

**Pull request merged**